test/unit/parser.test.js

```diff
@@ -9,6 +9,12 @@ describe('Standard Markdown', () => {
       expect(output).toEqual("<p>This is some text with <a href=\"http://example.com\">a link</a></p>\n");
     });
 
+    it('can render autolinks', () => {
+      const input = "This is some text that automatically links to <http://example.com>";
```

Comment:
Does the link need to be surrounded with `<` and `>` or should the following also work:

```js
const input = "This is some text that automatically links to http://example.com";
```

Reply:
Hmmmm, gross. Looks like right now, that would indeed also work because [GitHub-Flavored Markdown mode is enabled by default](https://github.com/remarkjs/remark/tree/master/packages/remark-parse#optionsgfm)

I think I'd like to turn that off; the only of those features that I think we want is tables (and perhaps strikethrough), and I think we specifically would like to avoid URL autolinking without `<>`. Thoughts?